simde/x86/avx2.h

```diff
@@ -2854,19 +2859,24 @@ simde_mm_maskload_epi64 (const int64_t mem_addr[HEDLEY_ARRAY_PARAM(2)], simde__m
     return _mm_maskload_epi64(HEDLEY_REINTERPRET_CAST(const long long *, mem_addr), mask);
   #else
     simde__m128i_private
-      mem_ = simde__m128i_to_private(simde_x_mm_loadu_epi64((mem_addr))),
```

Comment:
https://github.com/simd-everywhere/simde/actions/runs/4783302859/jobs/8505014399?pr=1004#step:9:25

```
../simde/x86/avx512/../avx2.h:2867:31: error: ‘mem_’ undeclared (first use in this function)
 2867 |       r_.neon_i64 = vandq_s64(mem_.neon_i64, vshrq_n_s64(mask_.neon_i64, 63));
      |                               ^~~~
```

Reply:
oh right :) thank you!